ios/Podfile.lock

```diff
@@ -343,7 +343,9 @@ PODS:
   - React-jsinspector (0.72.3)
   - React-logger (0.72.3):
     - glog
-  - react-native-safe-area-context (4.7.2):
```

Comment:
this updated podfile.lock is breaking build locally. Do we need this?

Reply:
@JeffGreiner-eaton I can run the showcase locally.